test/controllers/rooms_controller_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
```

Comment:
Acharia muito massa mesmo colocar um RSpec em tudo mas se for fazer isso não consigo cumprir com o prazo :(